gallery/src/styles/SelectableColors/SelectableColorsPage.tsx

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxP, NxCode, NxH3, NxTile } from '@sonatype/react-shared-components';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+
+import SelectableColorNxTagExample from './SelectableColorNxTagExample';
+import SelectableColorCustomExample from './SelectableColorCustomExample';
+
+const selectableColorNxTagExampleCode = require('./SelectableColorNxTagExample?raw'),
+    selectableColorCustomExampleCode = require('./SelectableColorCustomExample?raw'),
+    selectableColorCustomExampleStyles = require('./SelectableColorCustomExample.scss?raw');
+
+const customExampleCode =
+    [selectableColorCustomExampleCode, { content: selectableColorCustomExampleStyles, language: 'scss' }];
+
+const SelectableColorsPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <NxP>
+        RSC provides a collection of color values which are intended to be used anywhere that the user is intended to
+        be able to select from a small set of colors and anywhere that a collection of arbitrarily but distinctly
+        colored elements is needed. Examples of the former include tags and labels. Examples of the latter
+        include highlighted regions of a text document.
+      </NxP>
+      <NxP>
+        The colors that RSC provides for this purpose are known as the "selectable colors" and they are used within
+        the <NxCode>NxColorPicker</NxCode> and <NxCode>NxTag</NxCode> components as well as exposed publicly
+        for use in creating custom components.
+      </NxP>
+      <NxP>
+        For each named color within the selectable colors, two actual RGB color codes are provided, a lighter
+        variant and a darker one. This allows for two-tone components built with the selectable colors, as seen in
+        unselected selectable <NxCode>NxTag</NxCode>s.
```

Comment:
```suggestion
        the selected and unselected states of selectable <NxCode>NxTag</NxCode>s.
```

Reply:
That suggestion is less accurate.  It is specifically the unselected state of selectable tags which demonstrates this.

Reply:
> That suggestion is less accurate. It is specifically the unselected state of selectable tags which demonstrates this.

To me it's the fact that there are two states built with the selectable colours which is the point, but regardless the bigger issue with the language (to me) is that there are three uses of "selectable" in that sentence which makes it hard to parse.

Reply:
> To me it's the fact that there are two states built with the selectable colours which is the point, but regardless the bigger issue with the language (to me) is that there are three uses of "selectable" in that sentence which makes it hard to parse.

How's this: b4836612